.github/workflows/test.yml

```diff
@@ -78,3 +78,4 @@ jobs:
     - run: gem install bundler
     - run: bundle install --without=development
     - run: make test_git_rust
+
```

Comment:
These newline additions don't seem to belong here @syed-ghufran-hassan

Reply:
Sir I will check then